src/R/Interpreters/Impl/IMicrosoftRClientInstaller.cs

```diff
@@ -10,6 +11,6 @@ namespace Microsoft.R.Interpreters {
     /// of the Microsoft R Client. Exported via MEF.
     /// </summary>
     public interface IMicrosoftRClientInstaller {
-        void LaunchRClientSetup(ICoreShell coreShell, IFileDownloader downloader = null);
+        void LaunchRClientSetup(IServiceContainer coreShell, IFileDownloader downloader = null);
```

Comment:
This really should go to package and then Interpreters and broker are always x-plat